packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md

```diff
@@ -1,47 +1,54 @@
-The default:
+En komponent for å kunne søke igjennom en lenger liste med valg. Listen med valg har støtte for å vise flere atributter ved et element, for eksempel bedriftnavn og organisasjonsnummer.
+
+Om du vet du har 10 valg eller færre er det bedre å bruke en vanlig [dropdown](#dropdown).
+
+Om du er ute etter en måte å vise kontoinformasjon finnes en skreddersydd [kontovelger](#kontovelger).
```

Comment:
An aside - but should we use absolute URLs? This won't work when you're viewing a details page.